src/Plugin/media/Source/Media.php

```diff
@@ -742,4 +742,44 @@ public function doSaveThumbnailAsMedia() {
       !empty($configuration['media_image_entity_reference_field']);
   }
 
+  /**
+   * Get Mpx object id from uri string.
+   *
+   * @param string $mpx_object_id
+   *   The entity whose associated video will be checked to get the mpx id.
+   *
+   * @return int
+   *   The id from mpx object uri.
+   */
+  public static function getMpxObjectIdFromUri(string $mpx_object_id): int {
+    $global_url_parts = explode('/', $mpx_object_id);
+    $mpx_id = (int) end($global_url_parts);
+    return $mpx_id;
+  }
+
+  /**
+   * Get Mpx object id from entity.
+   *
+   * @param \Drupal\media\Entity\Media $entity
+   *   The entity whose associated video will be checked to get the mpx id.
+   *
+   * @return int
+   *   The id from mpx object related with the entity.
+   */
+  public static function getMpxObjectIdFromEntity(Media $entity): int {
```

Comment:
@germanmontesm Is this called from anywhere, or a leftover that needs removing?

Reply:
@salvamomo  this is helper function asked by @edutrul to be use if is necessary to grab the id.

Reply:
@germanmontesm Yes, I think it's useful to have a common utility like that. My question is around where it's used at the moment. I see `getMpxObjectIdFromUri` is called from the form class, but this one is not. We should try to avoid having code that is not used anywhere, specially if not tested. Thoughts @edutrul ?